### PR TITLE
Create Hello, GIT

### PR DESCRIPTION
**Pull request by riaba92**
Please merge me

### CODE REVIEW

**Comment by riaba92** [comment]
@annabratishko

**Comment by annabratishko** [comment]
@riaba92 All good

File: Hello, GIT [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Hello!
+
+Me name is Dima and I`m creating new file in Anya`s repository.
```
Comment (annabratishko) [inline Hello, GIT]: not Me

**Review by annabratishko — CHANGES_REQUESTED** [review]
Please review

File: Hello, GIT (hunk 1 above)
Comment (riaba92) [reply Hello, GIT]: Ok

Comment (annabratishko) [reply Hello, GIT]: correct please

**Review by annabratishko — CHANGES_REQUESTED** [review]

File: Hello, GIT (hunk 1 above)
Comment (riaba92) [reply Hello, GIT]: Ok

**Review by riaba92 — COMMENTED** [review]
done

**Pull request merged**